other/edge-server/client/README.md

```diff
@@ -1,38 +1,116 @@
 # Edge Server with MongoDB Node.js Driver
 
-This example application uses the MongoDB Node.js Driver to connect to 
-Edge Server. It has a React frontend for performing CRUD operations. It uses 
-the MongoDB Node.js Driver in an Express server application to connect to
-the Edge Server.
+This example todo application demonstrates using Edge Server with a MongoDB Driver. It uses the MongoDB Node.js Driver in an Express server application to connect to 
+Edge Server. It has a React frontend for performing CRUD operations. 
 
 React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App using the backend configuration provided in
-  this project's [TODO: Cory insert directory name here] directory.
+1. Create an App Services App that uses the configured backend provided in 
+  this project's ../backend directory.
 2. Download, configure, and start the Edge Server.
 3. Install dependencies and start the Node.js Express server.
 4. Install dependencies and start the React server.
 
-This is a MERN stack that connects to the Edge Server instead of Atlas. For
-a MERN stack tutorial, refer to 
+This is a MERN stack that connects to the Edge Server instead of Atlas. 
+(Note that MongoDB wire protocol connections connect directly to Edge Server 
+without authorization.) 
+
+For a MERN stack tutorial, refer to 
 [How to Use Mern Stack: A Complete Guide](https://www.mongodb.com/languages/mern-stack-tutorial).
 
 ## Create an App Services App
 
-TODO: Cory to write instructions for creating the backend using the provided
-dir.
+Edge Server connects to Atlas through an App Services App backend. 
+For this example application, you'll create an App Services App based on the 
+`edge-server.todo` template app. 
+
+This template app uses a backend with the following enabled: 
+
+- Device Sync configured with:
+  - An already defined `Item` collection in a `todo` database
+  - A single default rule that allows *any* user to read or write Items to 
+    the collection
+- Anonymous authentication only
```

Comment:
Hmm. I wonder if we can disable anonymous authentication and still use Edge Server? We might want to test that, and remove this line if it is possible. 

Mainly, I don't want to mislead people into thinking anonymous auth is important here somehow if it's not required. The wireprotocol connection doesn't use it, so I think if we do want to call out that anonymous authentication is configured - i.e. if it _is_ required - we have to say why.

(I _think_ it might have been enabled before because we were trying to make this interoperable with Device SDK clients, but we've found that with the required rules, we can't do that. So it might make sense to just turn it off.)

Reply:
Also, I wonder if we should explicitly call out that this differs from the Device SDK todo backend, and that this app won't be interoperable with Device SDK template apps.